path_config.go

```diff
@@ -149,6 +159,8 @@ type kubeConfig struct {
 	CACert string `json:"ca_cert"`
 	// TokenReviewJWT is the bearer to use during the TokenReview API call
 	TokenReviewerJWT string `json:"token_reviewer_jwt"`
+
+	Issuer string `json:"issuer"`
```

Comment:
We should add a code comment here similar to the other fields in the `kubeConfig` that details this is optional and a default assumed.